src/avt/VisWindow/Colleagues/VisWinRendering.C

```diff
@@ -1625,10 +1630,17 @@ VisWinRendering::ScreenRender(avtImageType imgT,
             unsigned char *rgbbuf = input->GetImage().GetRGBBuffer();
             int nchan = input->GetImage().GetNumberOfColorChannels();
 
+#if LIB_VERSION_LE(VTK,8,1,0)
```

Comment:
We removed VTK 8 support in develop.

Reply:
This is for the `3.4RC`. I already did `develop` and the VTK 8 support isn't in that commit.

Reply:
Apologies. Misread. Thought I already reviewed the 3.4RC PR (twice).